owncloudApp/src/androidTest/java/com/owncloud/android/settings/more/SettingsMoreFragmentTest.kt

```diff
@@ -325,6 +327,16 @@ class SettingsMoreFragmentTest {
         )
     }
 
+    @Test
+    fun feedbackOpensAlertDialogIfFeedbackMailIsEmpty() {
+        launchTest()
+        every { moreViewModel.getFeedbackMail() } returns EMPTY_VALUE
+
+        onView(withText(R.string.prefs_send_feedback)).perform(click())
+
+        onView(withText(R.string.drawer_feedback)).check(ViewAssertions.matches(isDisplayed()))
```

Comment:
Would be nice to test as well that the message appears, not only the title

Reply:
Added message too